src/components/EditableField/EditableField.tsx

```diff
@@ -225,71 +227,89 @@ export class EditableField extends React.Component<
 
   handleInputBlur = payload => {
     const { name, event } = payload
-    const {
-      activeField,
-      fieldValue,
-      initialFieldValue,
-      multipleValuesEnabled,
-      validationInfo,
-    } = this.state
-    const { validate, onCommit, onInputBlur } = this.props
+    const { activeField, multipleValuesEnabled } = this.state
+    const { validate, onCommit, onDiscard, onInputBlur } = this.props
 
-    if (equal(initialFieldValue, fieldValue)) {
+    if (equal(this.state.initialFieldValue, this.state.fieldValue)) {
```

Comment:
any reason why you moved away from destructuring the state object (just curious)

Reply:
I'm not sure how this change snuck in there :(